src/py/_pyodide/_core_docs.py

```diff
@@ -144,22 +145,30 @@ def to_js(
 
         Parameters
         ----------
-        obj : Any
-            The Python object to convert
-
-        depth : int, default=-1
-            The maximum depth to do the conversion. Negative numbers are treated
-            as infinite. Set this to 1 to do a shallow conversion.
-
-        pyproxies: JsProxy, default = None
-            Should be a Javascript ``Array``. If provided, any ``PyProxies`` generated
-            will be stored here. You can later use :any:`destroy_proxies` if you want
-            to destroy the proxies from Python (or from Javascript you can just iterate
-            over the ``Array`` and destroy the proxies).
-
-        create_pyproxies: bool, default=True
-            If you set this to False, :any:`to_js` will raise an error
-
+        obj : Any The Python object to convert
```

Comment:
Even, if this seems to currently render fine, there is really a spec for Python docstrings so it should be,
```
x : type
    Description of parameter `x`.
```
with optionally a default value, as it was before. Indentation value for the Description doesn't matter. https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
Oops, my text editor textwrap tool did this.

Reply:
This is why I had the blank lines in there before.